docs/developers/contributing.md

```diff
@@ -23,7 +23,14 @@ Set the `upstream` remote to the base `napari` repository:
 git remote add upstream https://github.com/napari/napari.git
 ```
 
-Install the package in editable mode, along with all of the developer tools
+Napari can be installed on most macOS, Linux, and Windows systems with Python 3.7, 3.8, and 3.9 using pip.  First, 
```

Comment:
We recently deprecated 3.7 support (see our package metadata [here](https://github.com/napari/napari/blob/bebc06f18e6c6675c9005210d2c63d5b82903b29/setup.cfg#L25-L27)).

```suggestion
Napari can be installed on most macOS, Linux, and Windows systems with Python 3.8, 3.9, and 3.10 using pip.  First, 
```